docs/source/notebooks/mmm/mmm_lift_test.ipynb

```diff
@@ -0,0 +1,1776 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "258ff2e5",
+   "metadata": {},
+   "source": [
+    "(mmm_lift_test)=\n",
+    "\n",
+    "# Lift Test Calibration\n",
+    "\n",
+    "## Introduction\n",
+    "\n",
+    "You may have heard of the phrase \"all models are wrong; some models are useful.\" This is true in many areas, and it's likely that after the first attempt, you haven't yet created a model that can accurately determine your true attribution. Even if you have, how can you be sure? We create models to understand the attribution of our marketing channels, but it appears that even then, we can't always rely on what the models tell us.\n",
+    "\n",
+    "In order to ensure that our models are decomposing correctly, we can use various testing methods to gather real-world data and compare it with our models. This will help us to identify any discrepancies and improve the decomposition accuracy of our models. \n",
+    "\n",
+    "Today, we will explore a new way to integrate experiments into **pymc-marketing**. This will bring us closer to accurate representations of real-world values and improve the estimates generated by our models.\n",
+    "\n",
+    "## Requirements\n",
+    "\n",
+    "Today, we won't be discussing how to conduct various tests, but instead, we will focus on their utilization. If you wish to acquire knowledge on how to generate results that are compatible with your MMM models, you can check out [CausalPy](https://causalpy.readthedocs.io/en/latest/) for conducting experiments.\n",
+    "\n",
+    "## Goal\n",
+    "\n",
+    "After reading this notebook, you will have gained the necessary expertise to incorporate the results (detected uplift from your experiments) into our regressive model.\n",
+    "\n",
+    "This notebook will display using the `add_lift_test_measurements` method of `DelayedSaturatedMMM` and its workflow:\n",
+    "\n",
+    "1. Build model: `mmm.build_model(X, y)`\n",
+    "2. Add lift measurements: `mmm.add_lift_test_measurements(df_lift_test)`\n",
+    "2. Sample posterior: `mmm.fit(X, y)`\n",
+    "\n",
+    "This in a case study of two correlated channels and see how lift tests help distinguish the channel effects."
```

Comment:
Typos.

Replace:

"This in a case study of two correlated channels and see how lift tests help distinguish the channel effects."

With:

"This is a case study of two correlated channels to see how lift tests help distinguish channel effects."